lib/rules/logical-assignment-operators.js

```diff
@@ -188,7 +188,6 @@ module.exports = {
             }]
         },
         fixable: "code",
-        // eslint-disable-next-line eslint-plugin/require-meta-has-suggestions -- Does not detect conditional suggestions
```

Comment:
This false positive has been fixed in the new version of `eslint-plugin/require-meta-has-suggestions`.